api/mutations/mutations.go

```diff
@@ -0,0 +1,71 @@
+package mutations
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/dgraph-io/dgo/v240/protos/api"
+	"github.com/dgraph-io/dgraph/v24/protos/pb"
+	"github.com/dgraph-io/dgraph/v24/schema"
+	"github.com/hypermodeinc/modusdb/api/utils"
+)
+
+func HandleReverseEdge(jsonName string, value reflect.Type, nsId uint64, sch *schema.ParsedSchema,
```

Comment:
we should move these to internal package if we want to have this package structure.